### PR TITLE
Refactored image file extensions function

### PR DESCRIPTION
**Pull request by jraval**
# Description

Refactored a function to support tif/tiff files and I made it more [DRY](https://www.wikiwand.com/en/Don%27t_repeat_yourself)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: data_management/autopreprocess_testing/preprocess_directory.py [hunk 1]
```diff
@@ -19,9 +19,10 @@
 target_dir = os.getcwd() + '/processed_test_images/'
 
 # Get list of all image paths
-jpg_files = glob.glob(source_dir + "/**/*.jpg", recursive=True)
-png_files = glob.glob(source_dir + "/**/*.png", recursive=True)
-img_files = jpg_files + png_files
+supported_file_extensions = ["jpg", "jpeg", "png", "tif", "tiff"]
```
Comment (bofori20) [inline data_management/autopreprocess_testing/preprocess_directory.py:22]: This is very helpful work, Jay. This refactored code should make it easier with preprocessing various types of data.

**Review by bofori20 — APPROVED** [review]
This is a very good refactored function. Should make our work moving forward much easier.

**Review by jacob-torres — COMMENTED** [review]
Nice work! Glad that you caught this, because it covers our bases for higher quality image files. It's a simple, elegant solution.

File: data_management/autopreprocess_testing/preprocess_directory.py (hunk 1 above)
Comment (ragerdude) [reply data_management/autopreprocess_testing/preprocess_directory.py:22]: Changing the code to be DRY using a for loop is really helpful. Thank you.

**Review by ragerdude — APPROVED** [review]

**Review by ashtilawat — CHANGES_REQUESTED** [review]
Can I get a quick loom video explaining this code?

**Review by ashtilawat — APPROVED** [review]

**Comment by jraval** [comment]
I added a loom link, here it is:

https://www.loom.com/share/3f4e599aa2e141ba8f160db5250169e4

**Comment by jraval** [comment]
[As a developer, I want to increase the accuracy of the Tesseract model by finding the best preprocessing steps.](https://trello.com/c/3PndWaDa/848-as-a-developer-i-want-to-increase-the-accuracy-of-the-tesseract-model-by-finding-the-best-preprocessing-steps)

**Comment by derekjpeters** [comment]
[Created function for adding image files to folder with Niger's help.](https://trello.com/c/T3Tn5C6b/866-created-function-for-adding-image-files-to-folder-with-nigers-help)

**Pull request merged**